bundle.yaml.j2

```diff
@@ -37,26 +37,25 @@ applications:
     series: jammy
     scale: 1
     trust: true
-  tls-certificates-operator:
-    charm: tls-certificates-operator
-    channel: stable
+  self-signed-certificates:
+    charm: self-signed-certificates
+    channel: edge
     scale: 1
     {%- if testing %}
     options:
       ca-common-name: demo.ca.local
-      generate-self-signed-certificates: true
     {%- endif %}
   traefik-admin:
     charm: traefik-k8s
-    revision: 136
-    channel: edge
+    revision: 143
+    channel: latest/edge
```

Comment:
Question: Why do we need to specify both the revision and the channel? Shouldn't the revision be enough

Reply:
Sometime this week Shu raised the point to pin the charms in the bundle to revisions that we know are working correctly. I didn't think it's an issue to also include the channels. I'm not against taking the channel specification out.

Reply:
If I understand the concepts of juju charm channel and revision correctly, they are not exclusive to each other. (pls correct me if I'm wrong). Also, refer to this [documentation](https://juju.is/docs/juju/bundle).

```
    #
    # revision:
    #
    # States the revision of the charm should be used when deploying a
    # non-local charm. Use requires a channel to be specified, indicating
    # which channel should be used when refreshing the charm. 
    #

    revision: 8

    #
    # channel:
    #
    # States what the preferred channel should be used when deploying a
    # non-local charm.
    #  - Charmhub charms expect <track>/<risk>/<branch> format (latest/stable).
    #

    channel: latest/edge
```

Reply:
I edited my previous comment. I rather we include both the channel and the revision.